var/spack/repos/builtin/packages/zlib/package.py

```diff
@@ -33,6 +33,7 @@ class Zlib(AutotoolsPackage):
     homepage = "http://zlib.net"
     url = "http://zlib.net/zlib-1.2.8.tar.gz"
 
+    version('1.2.10', 'd9794246f853d15ce0fcbf79b9a3cf13')
     version('1.2.8', '44d667c142d7cda120332623eab69f40')
```

Comment:
Could you please change the old line to:
```
version('1.2.8', '44d667c142d7cda120332623eab69f40',
    url='http://pkgs.fedoraproject.org/repo/pkgs/mingw-zlib/zlib-1.2.8.tar.gz/44d667c142d7cda120332623eab69f40/zlib-1.2.8.tar.gz')
```
This will ensure that the old version still works, even though it's been removed from the `zlib` website.  And because the tarball from Fedora has the same checksum (I checked it), we know it's the same that we've been working with all along.